test/unit/discovered_extensions_test.rb

```diff
@@ -4,6 +4,8 @@ class FindDiscoveryRulesTest < ActiveSupport::TestCase
   include Foreman::Controller::DiscoveredExtensions
 
   setup do
+    SETTINGS[:organizations_enabled] = true
```

Comment:
This isn't needed in other tests, we always test with taxonomies on and don't guarantee tests will pass with taxonomies off afaik. Tests pass on the latest branch for both discovery and foreman without this, any ideas to reproduce the failure?